### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by TediJaho** [jasp-stats/jasp-issues#2429]
### Purpose

Feature request

### Is your feature request related to a problem?

It takes a while to remove all filters

### Describe the solution you would like

Have the ability to deselect all values at once when filtering

### Additional context

IS it possible to have an option to deselect all input values when filtering words? At our office we categorize data by distinct words and we might have 60 or so different values i want to count. Jasp allows me to remove them one by one but if i need 3 for example i will have to remove 57 first. and that usually lags a bit.

**Comment by tomtomme** [comment]
Would be cool!
Maybe add a checkbox that says "select all" and when thats unchecked it deselects all filters from categorial vars.

**Comment by TediJaho** [comment]
Yes! Jasp has been great so far for me. The only issue I seem to be having nowadays with it is overall filter options while preparing the data.

**Comment by tomtomme** [comment]
@TediJaho 
Can you please rename the title of this issue to 

"[Feature Request]: Filter Categorial Vars with Select/Unselect all checkbox"

**Comment by TediJaho** [comment]
Hi Thomas, it's done

**Comment by tomtomme** [comment]
@TediJaho 
for your information: a "reset all filters" button is now available, but not yet the "unselect" button
